lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1157,14 +1157,13 @@ def _create_node_runbook(
             # the max length of vm name is 64 chars. Below logic takes last 40
             # chars in resource group name and keep the leading "lisa-". So it's
             # easy to identify it's a lisa created node.
-            azure_node_runbook.name = truncate_keep_prefix(
-                f"{name_prefix}-n{index}", 50
-            )
-            # It's used as computer name only. Windows doesn't support name more
-            # than 15 chars
-            azure_node_runbook.short_name = truncate_keep_prefix(
-                azure_node_runbook.name, 15
-            )
+            node_name = f"{name_prefix}-n{index}"
+            azure_node_runbook.name = truncate_keep_prefix(node_name, 50, node_name[:5])
```

Comment:
Add comments above that, the leading "lisa-" is handled in previous logic. The name_prefix includes both customized and generated. So skip the validation.

Reply:
Added comment.

Reply:
The description doesn't specify the reason actually. There is validation to make sure either it's existing rg name, or start with "lisa-". Because the two conditions are handled before calling into this method, and it cannot be differentiated here. So, we just use first 5 chars to skip the head validation.

Reply:
If we don't specify clearly, the logic will be confused, and in future, it may be broken by others.